src/execute/ExecuteView.tsx

```diff
@@ -1,29 +1,26 @@
-import React, { useState, useRef, useEffect } from "react";
+import React, { useState, useEffect } from "react";
```

Comment:
Do we need useEffect?

Reply:
I think so, We are using it to sync local exerciseXML state with db's exercise.